### PR TITLE
Contact us Send button fixed

### PR DESCRIPTION
**Pull request by deepakpathania-zz**
The contact us button which was not properly aligned on smaller screen sizes has been fixed.

### CODE REVIEW

**Comment by handasolo** [comment]
We don't accept Pull Request on master branch. Please, make one on develop branch.

**Comment by CuriousLearner** [comment]
Hello @deepakpathania Thanks for your work, but we accept all PR in develop branch only and not directly in master.

We're in process of having the doc for this in place, and sorry that it caused you trouble. Please do the changes, and submit another one.

File: index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+
```
Comment (handasolo) [inline index.html:26]: Also no need to add a space here.

File: index.html [hunk 2]
```diff
@@ -386,7 +387,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
                   <div class="form_group">
                     <div class="row">
                       <span id="success" class="col-xs-4"></span>
-                      <input type="submit" name="send" id="send" value="Send" class="col-sm-8 btn btn-lg send-button center-block" >
+
```
Comment (handasolo) [inline index.html:390]: and here.

File: index.html [hunk 3]
```diff
@@ -386,7 +387,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
                   <div class="form_group">
                     <div class="row">
                       <span id="success" class="col-xs-4"></span>
-                      <input type="submit" name="send" id="send" value="Send" class="col-sm-8 btn btn-lg send-button center-block" >
+                      
+                      <input type="submit" name="send" id="send" value="Send" class="col-sm-8 col-xs-pull-2 col-xs-push-1.5 col-sm-push-0 col-sm-pull-0 col-lg-offset-2 btn btn-lg send-button center-block" >
+
```
Comment (handasolo) [inline index.html:392]: and here as well.

**Pull request closed without merging**